lib/src/components/NxIconDropdown/NxIconDropdown.tsx

```diff
@@ -13,27 +13,20 @@ import NxButton from '../NxButton/NxButton';
 import NxFontAwesomeIcon from '../NxFontAwesomeIcon/NxFontAwesomeIcon';
 import './NxIconDropdown.scss';
 import NxOverflowTooltip from '../NxTooltip/NxOverflowTooltip';
-import NxDropdownMenu from '../NxDropdownMenu/NxDropdownMenu';
-import useDropdownEvents from '../../util/useDropdownEvents';
 
-const NxIconDropdown: FunctionComponent<Props> = function NxIconDropdown(props) {
+import AbstractDropdown, { AbstractDropdownToggleElement } from '../NxDropdown/AbstractDropdown';
+
+const NxIconDropdown: FC<Props> = function NxIconDropdown(props) {
```

Comment:
Why the change from `FunctionComponent` to `FC`?

Reply:
I think, when I was playing around with the code, it went over the max-len and I changed it to FC to remove the error temporarily but forgot to change it back or something like that. I changed it back here: a3e9a119a3904ddcecaa7ab82bcc38bcc6ccfcd3